### DISCUSSION TITLE
Make friends with WebXR and stellarium-web-engine

### DISCUSSION THREAD

**Discussion by TheEag1eS0n** [Stellarium/stellarium#2482] [General]
Hello everybody!

I'm trying to make friends with WebXR technology and stellarium-web-engine

How can I get an already rendered canvas with the image of stars and how?

Is there documentation or API for stellarium-web-engine?

**Comment by gzotti** [comment]
Please use the dedicated issues tracker of the stellarium-web-engine repository. https://github.com/Stellarium/stellarium-web-engine/issues
